specification/monitor/resource-manager/Microsoft.Insights/stable/2019-06-01/actionGroups_API.json

```diff
@@ -0,0 +1,879 @@
+                        "description": "The action group does not exist. It may have already been deleted."
+                    },
+                    "default": {
+                        "description": "An error occurred and the action group could not be deleted.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            },
+            "patch": {
+                "description": "Updates an existing action group's tags. To update other fields use the CreateOrUpdate method.",
+                "operationId": "ActionGroups_Update",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ActionGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    },
+                    {
+                        "name": "actionGroupPatch",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/ActionGroupPatchBody"
+                        },
+                        "description": "Parameters supplied to the operation."
+                    }
+                ],
+                "responses": {
+                    "default": {
+                        "description": "Error response describing why the operation failed.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    },
+                    "200": {
+                        "description": "An existing action group was successfully updated.",
+                        "schema": {
+                            "$ref": "#/definitions/ActionGroupResource"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "Patch an action group": {
+                        "$ref": "./examples/patchActionGroup.json"
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/providers/microsoft.insights/actionGroups": {
+            "get": {
+                "description": "Get a list of all action groups in a subscription.",
+                "operationId": "ActionGroups_ListBySubscriptionId",
+                "x-ms-examples": {
+                    "List action groups": {
+                        "$ref": "./examples/listActionGroups.json"
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": null
```

Comment:
>null [](start = 36, length = 4)

You should set this to "nextLink" even if your service only returns 1 page. Doing so will make adding paging easier for you in the future.